src/themes/default/pages/Home.vue

```diff
@@ -70,9 +66,6 @@ export default {
     everythingNewCollection () {
       return this.$store.state.homepage.new_collection
```

Comment:
would be perfect to add a getter for newCollection in store and use it here:)

Reply:
Right :+1:  I guess I can refactor that part as well.